lib/atscppapi/src/Transaction.cc

```diff
@@ -98,6 +97,46 @@ Transaction::~Transaction()
   delete state_;
 }
 
+bool Transaction::configIntSet(TSOverridableConfigKey conf, int value) 
+{
+    //txn,conf,value
+    TSReturnCode res=TSHttpTxnConfigIntSet(state_->txn_,conf,(TSMgmtInt)value);
+    return (res==TS_SUCCESS);
```

Comment:
I followed the way Transaction::setCacheUrl(const string &cache_url) is written in Transaction.cc. Anyway, I will make it concise. 

bool
Transaction::setCacheUrl(const string &cache_url)
{
  TSReturnCode res = TSCacheUrlSet(state_->txn_, cache_url.c_str(), cache_url.length());
  return (res == TS_SUCCESS);
}